cmd/chart-repo-sync/deleteRepo.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
file should just be called `delete.go`

Reply:
it's going to be unclear if you just use `delete.go` under folder `chart-repo-sync` as it's confusing what is going to be deleted with that function. If you want to use `delete.go` then I suggest we should rename `chart-repo-sync` to `app-repository` (the entity managed by apprepository-controller). Then we have `delete.go` and `sync.go` (and & sync)

Reply:
yeah I think now that this does more than just sync, renaming it to `chart-repo` would be good. Note, it shouldn't be app-repository because this cmd is specific to Helm repos.

Main thing is that the lower camel case is not an idiomatic convention in Go, so if you want to still call this deleteRepo, you can use `deleterepo` or `delete_repo` instead.

Reply:
Oh surprise... I am seeing lower camel case commonly used in Go. Probably it's not applied in file name. Btw it isn't a big thing. So let's rename to `chart-repo` and `delete.go`, `sync.go`. Adding a follow-up commit.

Reply:
yeah strictly talking about filenames here, of course camel case is used in function/variable naming.